submission/proxy_server.go

```diff
@@ -81,7 +80,11 @@ func (s *ProxyServer) HandleAddPreChain() http.HandlerFunc {
 			return
 		}
 		w.WriteHeader(http.StatusOK)
```

Comment:
Please put this line after all error cases.

Reply:
Done